base/docs/basedocs.jl

```diff
@@ -309,52 +301,52 @@ globals.
 kw"const"
 
 """
-Functions are defined with the `function` keyword:
+    function
 
-    function add(a, b)
-        return a + b
-    end
+Functions are defined with the `function` keyword:
 
+```julia
+function add(a, b)
+    return a + b
+end
+```
 Or the short form notation:
 
-    add(a, b) = a + b
-
-The use of the `return` keyword is exactly the same as in other languages, but is often
-optional. When it's not used, the last expression in the function body will be returned
-by default:
```

Comment:
This text seems important to retain.

Reply:
Just didn't feel relevant to the `function` docstring, and the same example is used in the `return` docstring. Thoughts?

Reply:
I mean, I guess it isn't directly relevant to the `function` keyword, but I'm not sure where else it would be appropriate to mention that. It's definitely worth noting somewhere obvious.

Reply:
Is this documented elsewhere?

Reply:
L319-L321